uk.co.compucorp.civicrm.tasksassignments/CRM/Tasksassignments/Reminder.php

```diff
@@ -131,14 +131,19 @@ private static function _extractPreviousAssignee(&$assignees, $previousAssigneeI
    * Gets the full list of the recipients of the reminder email
    * Makes sure there are no duplicates or no empty emails in the list
    *
-   * @param array $activityContacts
+   * @param array $contacts
    * @param array $previousAssignee
    * @return array
    */
-  private static function _reminderRecipients($activityContacts, $previousAssignee) {
-    return array_filter(array_unique(array_merge(
-                            $activityContacts['assignees']['emails'], $activityContacts['source']['emails'], [$previousAssignee['email']]
-    )));
+  private static function _reminderRecipients($contacts, $previousAssignee) {
+    $hasSources = !empty($contacts['source']['emails']);
```

Comment:
ops , I though php is smart enough to not show a notice for such case but I was wrong .. I suggest to use what I've suggest but instead of using short ternary operator syntax use the long one and add !empty() to check if the value is their.